lib/services/data_deletion.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+Future<void> deleteData(String id) async {
```

Comment:
make one class with the name TaskProvider / TaskServices and handle all data related stuff there